SU-SWS/stanford_profile_helper
# D8CORE-3142: Styling some padding for the su-intro

# READY FOR REVIEW

# Summary
- Adding spacing at the bottom of the new info block on Events, News and People

# Needed By (Date)
- 1/3/21

# Urgency
- Med

# Steps to Test

1. Pull in this and the other stanford_profile change.
2. Clear caches
3. Checkout the padding on the new intro content on the New, People, Events List page. 
4. Try adding some content. We will be writing some documentation on best practices since it is currently an open slate. 
5. Try removing the content and see that there is not extra space above the list. 

# Affected Projects or Products
- Stanford Basic and  SOE

# Associated Issues and/or People
- D8CORE-3142
- https://github.com/SU-SWS/stanford_profile/pull/345

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
